packages/cli/src/lib/project/project.ts

```diff
@@ -0,0 +1,57 @@
+import {createWriteStream, existsSync, unlinkSync} from 'fs';
+import {join} from 'path';
+import {cli} from 'cli-ux';
+import * as archiver from 'archiver';
+
+export class Project {
+  constructor(private pathToProject: string) {
+    this.ensureProjectCompliance();
+  }
+
+  private get pathToTemporaryZip() {
+    return join(this.pathToProject, 'snapshot.zip');
+  }
+
+  private get pathToResources() {
+    return join(this.pathToProject, 'resources');
+  }
+
+  deleteTemporaryZipFile() {
```

Comment:
Similar to another comment in your other PR, I'd prefer to have the `public` keyword explicitly set, and to have all public methods grouped at the beginning of the class.